inspire/modules/workflows/tasks/submission.py

```diff
@@ -93,9 +93,12 @@ def _create_ticket(obj, eng):
 
         # Trick to prepare ticket body
         body = "\n ".join([line.strip() for line in body.split("\n")])
-        rt = get_instance()
-        if not rt:
+        rt = None
+        if cfg.get("PRODUCTION_MODE"):
+            rt = get_instance()
+        if not rt or not cfg.get("PRODUCTION_MODE"):
```

Comment:
True, I added the `or not cfg.get("PRODUCTION_MODE")` before the `rt = None` and then forgot to take it. ^^